### DISCUSSION TITLE
why the precision of the LiSi example is very low?

### DISCUSSION THREAD

**Discussion by sunlight905** [deepmodeling/dpgen#1403] [Q&A]
I test the example LiSi [https://zhuanlan.zhihu.com/p/555628454,](https://zhuanlan.zhihu.com/p/555628454) all the parameters is unchanged, but when the training complete, the precision is very low?
Could you give me some advice?
Thanks!
2023-11-07 19:41:16,655 - INFO : system 000 candidate :    228 in  20100   1.13 %
2023-11-07 19:41:16,655 - INFO : system 000 failed    :  19352 in  20100  96.28 %
2023-11-07 19:41:16,655 - INFO : system 000 accurate  :    520 in  20100   2.59 %
2023-11-07 19:41:16,664 - INFO : system 000 accurate_ratio:   0.0259    thresholds: 0.9970 and 0.9990   eff. task min and max   -1   50   number of fp tasks:     50
2023-11-08 03:51:02,648 - INFO : system 000 candidate :    516 in  20100   2.57 %
2023-11-08 03:51:02,648 - INFO : system 000 failed    :  18523 in  20100  92.15 %
2023-11-08 03:51:02,648 - INFO : system 000 accurate  :   1061 in  20100   5.28 %
2023-11-08 03:51:02,656 - INFO : system 000 accurate_ratio:   0.0528    thresholds: 0.9970 and 0.9990   eff. task min and max   -1   50   number of fp tasks:     50
2023-11-08 13:47:30,097 - INFO : system 000 candidate :    525 in  30100   1.74 %
2023-11-08 13:47:30,097 - INFO : system 000 failed    :  28498 in  30100  94.68 %
2023-11-08 13:47:30,097 - INFO : system 000 accurate  :   1077 in  30100   3.58 %
2023-11-08 13:47:30,110 - INFO : system 000 accurate_ratio:   0.0358    thresholds: 0.9970 and 0.9990   eff. task min and max   -1   50   number of fp tasks:     50
2023-11-08 23:13:54,374 - INFO : system 000 candidate :    853 in  30100   2.83 %
2023-11-08 23:13:54,374 - INFO : system 000 failed    :  27764 in  30100  92.24 %
2023-11-08 23:13:54,374 - INFO : system 000 accurate  :   1483 in  30100   4.93 %
2023-11-08 23:13:54,387 - INFO : system 000 accurate_ratio:   0.0493    thresholds: 0.9970 and 0.9990   eff. task min and max   -1   50   number of fp tasks:     50
2023-11-09 10:07:28,022 - INFO : system 000 candidate :    691 in  25100   2.75 %
2023-11-09 10:07:28,022 - INFO : system 000 failed    :  23258 in  25100  92.66 %
2023-11-09 10:07:28,022 - INFO : system 000 accurate  :   1151 in  25100   4.59 %
2023-11-09 10:07:28,033 - INFO : system 000 accurate_ratio:   0.0459    thresholds: 0.9970 and 0.9990   eff. task min and max   -1   50   number of fp tasks:     50
2023-11-09 23:50:19,391 - INFO : system 000 candidate :   1567 in  40100   3.91 %
2023-11-09 23:50:19,391 - INFO : system 000 failed    :  36697 in  40100  91.51 %
2023-11-09 23:50:19,391 - INFO : system 000 accurate  :   1836 in  40100   4.58 %
2023-11-09 23:50:19,408 - INFO : system 000 accurate_ratio:   0.0458    thresholds: 0.9970 and 0.9990   eff. task min and max   -1   50   number of fp tasks:     50

`{
    "type_map": ["Li", "Si"],
    "mass_map": [6.9, 28.1],
    "init_data_prefix": "",
    "init_data_sys": [
       "POSCAR_LiSi.01x01x01/02.md/sys-0016-0016/deepmd"
    ],    
    "sys_configs_prefix": "",
    "sys_configs": [
      ["POSCAR_LiSi.01x01x01/02.md/sys-0016-0016/scale-1.000/00000[0-2]/POSCAR"]
    ],

    "_comment": " 00.train ",
    "numb_models": 4,
    "model_devi_activation_func" :        [["tanh","tanh"],["tanh","gelu"],["gelu","tanh"],["gelu","gelu"]],
      
    "default_training_param": {
       "model": {
              "type_map": ["Li", "Si"],
              "descriptor": {
                  "type": "se_e2_a",
                  "sel": [100, 100],
                  "rcut_smth": 2.0,
                  "rcut": 6.0,
                  "neuron":[25, 50, 100],
                  "resnet_dt": false,
                  "type_one_side": true,
                  "axis_neuron": 12,
                  "seed": 1
              },
              "fitting_net": {
                  "neuron": [240, 240, 240],
                  "resnet_dt": true,
                  "seed": 1
              }
          },
          "learning_rate": {
              "type": "exp",
              "start_lr": 0.001,
              "decay_steps": 4000
          },
          "loss": {
              "start_pref_e": 0.02,
              "limit_pref_e": 1,
              "start_pref_f": 1000,
              "limit_pref_f": 1,
              "start_pref_v": 0.0,
              "limit_pref_v": 0.0
          },
          "training": {
              "stop_batch": 800000,
              "disp_file": "lcurve.out",
              "disp_freq": 2000,
              "numb_test": 4,
              "save_freq": 2000,
              "save_ckpt": "model.ckpt",
              "disp_training": true,
              "time_training": true,
              "profiling": false,
              "profiling_file": "timeline.json",
              "_comment": "that's all"
          }
    },
      
    "_comment": " 01.model_devi ",
    "model_devi_dt": 0.001,
    "model_devi_skip": 0,
    "model_devi_f_trust_lo": 0.15,
    "model_devi_f_trust_hi": 0.35,
    "model_devi_clean_traj": false,
    "model_devi_jobs": [
       {"sys_idx": [0],"temps": [200, 300, 400, 500],"press": [1.0, 100.0, 1000.0, 10000.0, 50000.0], "trj_freq": 5, "nsteps":   1000, "ensemble": "npt", "_idx":     "00" },
       {"sys_idx": [0],"temps": [200, 300, 400, 500],"press": [1.0, 100.0, 1000.0, 10000.0, 50000.0], "trj_freq": 5, "nsteps":   1000, "ensemble": "npt", "_idx":     "01" },
       {"sys_idx": [0],"temps": [200, 300, 400, 500],"press": [1.0, 100.0, 1000.0, 10000.0, 50000.0], "trj_freq": 10, "nsteps":   3000, "ensemble": "npt", "_idx":     "02" },
       {"sys_idx": [0],"temps": [200, 300, 400, 500],"press": [1.0, 100.0, 1000.0, 10000.0, 50000.0], "trj_freq": 10, "nsteps":   3000, "ensemble": "npt", "_idx":     "03" },
       {"sys_idx": [0],"temps": [200, 300, 400, 500],"press": [1.0, 100.0, 1000.0, 10000.0, 50000.0], "trj_freq": 20, "nsteps":   5000, "ensemble": "npt", "_idx":     "04" },
       {"sys_idx": [0],"temps": [200, 300, 400, 500],"press": [1.0, 100.0, 1000.0, 10000.0, 50000.0], "trj_freq": 20, "nsteps":   8000, "ensemble": "npt", "_idx":     "05" }
    ],
      
    "_comment": " 02.fp ",
    "fp_style": "vasp",
    "fp_accurate_threshold": 0.999,
    "fp_accurate_soft_threshold": 0.997,
    "shuffle_poscar": false,
    "fp_task_max": 50,
    "fp_task_min": 3,
    "ratio_failed": 0.05,
    "fp_pp_path": "<your-POTCAR-path>",
    "fp_pp_files": [
      "POTCAR_Li","POTCAR_Si"
    ],
    "fp_incar": "./INCAR_LiSi"
  }   `
